website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/reference/rust-sdk-guide.md

```diff
@@ -0,0 +1,699 @@
+---
+title: 'Stylus Rust SDKオーバービュー'
+sidebar_label: 'Stylus Rust SDKオーバービュー'
+description: 'An in-depth overview of the features provided by the Stylus Rust SDK'
+author: rachel-bousfield
+translator: Ying@Fracton, Naoki@Fracton, Hikaru@Fracton
+sme: rachel-bousfield
+sidebar_position: 1
+target_audience: Developers using the Stylus Rust SDK to write and deploy smart contracts.
+---
+
+import PublicPreviewBannerPartial from '../partials/_stylus-public-preview-banner.md';
```

Comment:
There is no file named `_stylus-public-preview-banner.md`, you mean `_stylus-public-preview-banner.md_partial.md`?